apps/omg_watcher_info/test/fixtures.exs

```diff
@@ -142,25 +147,21 @@ defmodule OMG.WatcherInfo.Fixtures do
   end
 
   defp prepare_one_block({blknum, recovered_txs}) do
-    mined_block = %{
+    block_application = %{
       transactions: recovered_txs,
-      blknum: blknum,
-      blkhash: "##{blknum}",
+      number: blknum,
+      hash: "##{blknum}",
```

Comment:
is this ok?

Reply:
it seems it always was that way.
by git blaming, I found this pr  - https://github.com/omgnetwork/elixir-omg/pull/1123